### DISCUSSION TITLE
Modification du reflet de la loi d'eau

### DISCUSSION THREAD

**Discussion by Guillaume31a** [witsa/synapps#587] [Q&A]
Bonjour,

Je souhaiterai modifier le reflet de la loi d'eau à ma guise avec un code HTML que j'incrémenterais dans le stack où elle se situe.

Il permettrait d'influer sur la disposition des points, la couleur, la taille etc...  enfin bref, sur tout !

Je n'ai malheureusement pas encore assez de compétences en programmation pour retrouver le code entier qui crée et configure ma lois d'eau.

Je me redirige donc vers vous afin d'obtenir de l'aide,

PS : voici le reflet concerné 
![image](https://user-images.githubusercontent.com/131860668/236165452-e9e6ee7e-a4a4-4edc-ba42-d93b082a0296.png)

![image](https://user-images.githubusercontent.com/131860668/236165673-b5555438-d1e5-4f8c-ba66-7744725ac5cd.png)

Merci d'avance pour votre réponse.

**Comment by manufitoussiwit** [comment]
Salut!
Pour réaliser ceci, il faut faire appel à des notions de programmation avancées qu'il seraient difficiles à partager ici malheureusement.
En l'occurrence, il faut apprendre à consommer la donnée de l'acteur détails de reflet qui contient les coordonnées et les utiliser pour configurer un nouveau graphique créé avec la librairie ChartJS.

Aussi, je pense que vous devriez envisager de suivre les formations Synapps et notamment la plus avancée d'entre elles : "Expert". Elle permet de savoir comment mettre à profit ses compétences en développement WEB dans Synapps. Elle est accessible après avoir suivi les deux premières : "Découvreur" et "Créateur".

**Reply by Guillaume31a** [reply]
Je vais chercher de mon coté, 
Merci pour votre réponse, très bonne journée à vous.
